scripts/who/gho/svNames/README.md

```diff
@@ -0,0 +1,14 @@
+## To generate names for WHO stat vars
+1. This script reads the who_properties.mcf and the who_gho_stat_vars.mcf files that are checked into the schema (and google3) repo. To copy these files over:
+  ```
+  WHO_PROPS_DIR=<directory-of-g4-client>/third_party/datacommons/schema/core
```

Comment:
can we instead link to the schema repo here? we shouldn't add google3 details to the repo.

Reply:
- added test
- the who_properties.mcf file is in the schema repo so I updated the link there. However, the who_gho_stat_vars.mcf file is not in the schema repo, so I could not update the link there ..

Reply:
sg